dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.40/V2_40_5__add_evenhook_table.sql

```diff
@@ -5,6 +5,7 @@ create table if not exists eventhook (
     eventhookid int8 not null,
     uid varchar(11) null,
     code varchar(100) null,
+	disabled bool NOT NULL,
```

Comment:
Remove extra indentation, probably tabs instead of spaces.

Reply:
That just a diff tab.. there are no extra spaces in the file itself